### DISCUSSION TITLE
Why are individual routes deactivated as default?

### DISCUSSION THREAD

**Discussion by fm-sys** [cgeo/cgeo#9615] [Q&A] [answered]
Individual routes and UDC's are two very nice features which c:geo has implemented in the recent past. But unfortunately it is disabled per default, so most users will never notice and use it.

Why are long click features deactivated as default or even is there a reason why it has to be configurable at all?

![Screenshot_20201228_121702_cgeo geocaching_copy_431x189](https://user-images.githubusercontent.com/64581222/103211206-d6aa0f00-4907-11eb-9928-f1ac32c6dd38.png)

**Comment by Lineflyer** [comment]
AFAIR this was done after beta testers reported that they created an individual route / UDC accidentally and did not know what it is and how to undo it.

**Reply by fm-sys** [reply]
IMHO we should rather think about how to better explain new features instead of hiding them from mainstream users... ;-)

**Reply by moving-bits** [reply]
As we already have a setting for it, we could change the default to "activated" and do a one-time explanation for it.
So users get to know it, but have the chance to disable it if they don't need it or don't want it.

What do you think?

**Reply by Lineflyer** [reply]
Why not activating it by default. But where (or on what action) would you place a one-time dialog?

**Reply by moving-bits** [reply]
one-time dialog could be displayed on opening a map, like we do with the info on quick map settings.

**Reply by moving-bits** [reply]
I'll create a PR for it

**Reply by moving-bits** [reply]
Implemented with #9667

**Comment by moving-bits** [comment, marked as answer]
Is there a way to mark a discussion as resolved or closed?

**Reply by Lineflyer** [reply]
In case its a "Q&A" you can mark it answered. Done :)

**Reply by moving-bits** [reply]
I had tried to do this here, but it seems I can do that only on discussions I have started...
